recipe/conda_build_config.yaml

```diff
@@ -21,5 +21,5 @@ pin_run_as_build:
   boost-cpp:
     max_pin: x.x.x
 docker_image:                                  # [linux64]
- - quay.io/condaforge/linux-anvil-comp7        # [linux64]
+ - quay.io/condaforge/linux-anvil-cos7-x86_64  # [linux64]
```

Comment:
For the record, this fixes:
```
ValueError: variant config in /home/pearu/git/pearu/omniscidb-feedstock/recipe/conda_build_config.yaml is ambiguous because it
does not fully implement all zipped keys, or specifies a
subspace that is not fully implemented. To be clear:
.. we did not find ['quay.io/condaforge/linux-anvil-comp7'] from {'c_compiler': ['gcc'], 'c_compiler_version': ['9'], 'cxx_compiler': ['gxx'], 'cxx_compiler_version': ['9'], 'cgo_compiler': ['go-cgo'], 'cgo_compiler_version': ['1.15'], 'cuda_compiler': ['nvcc'], 'cuda_compiler_version': ['None', '11.0'], 'pin_run_as_build': {'boost-cpp': {'max_pin': 'x.x.x'}}, 'docker_image': ['quay.io/condaforge/linux-anvil-comp7', 'quay.io/condaforge/linux-anvil-cuda:11.0']} in docker_image:['quay.io/condaforge/linux-anvil-cos7-x86_64', 'quay.io/condaforge/linux-anvil-cos7-cuda:10.2', 'quay.io/condaforge/linux-anvil-cuda:11.0', 'quay.io/condaforge/linux-anvil-cuda:11.1', 'quay.io/condaforge/linux-anvil-cuda:11.2']
```